pkgs/by-name/go/gogh/package.nix

```
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  ncurses,
+  python3,
+}:
+
+stdenv.mkDerivation (
+  finalAttrs:
+  let
+    pythonEnv = python3.withPackages (
+      ps: with ps; [
+        ruamel-yaml
+        unidecode
+        pyyaml
+        tomli
+        tomli-w
+        configobj
+      ]
+    );
+  in
+  {
+    pname = "gogh";
+    version = "361";
+
+    src = fetchFromGitHub {
+      owner = "Gogh-Co";
+      repo = "Gogh";
+      tag = "v${finalAttrs.version}";
+      hash = "sha256-kzJiI1w8CYVYMcHo8mMqLseyFsU/PekjMkkk0gG+RH4=";
+    };
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir --parents $out/bin/
+      cp gogh.sh $out/bin/${finalAttrs.meta.mainProgram}
+      cp *.py $out/bin/
+      cp apply-colors.sh $out/bin/
+      cp -r installs themes $out/bin/
+      wrapProgram $out/bin/${finalAttrs.meta.mainProgram} \
+        --prefix PYTHONPATH : "${pythonEnv}/${pythonEnv.sitePackages}" \
+        --prefix PATH : "${pythonEnv}/bin"
+
+      runHook postInstall
+    '';
+
+    strictDeps = true;
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    buildInputs = [ ncurses ];
+
+    meta = {
+      description = "Collection of color schemes for various terminal emulators, including Gnome Terminal, Pantheon Terminal, Tilix, XFCE4 Terminal and iTerm on macOS";
```

this description is kinda long, maybe save listing the support terminals for the long description?
```suggestion
      description = "Collection of color schemes for various terminal emulators";
```